From e8253c6513e604bb2a4aa9de18682b6f96eeef25 Mon Sep 17 00:00:00 2001
From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Sun, 10 Apr 2022 16:16:00 +0200
Subject: [PATCH] ssml: Use a larger buffer

When building espeak-ng in a directory that has a long path, 80 bytes to
store it is quite short.
---
 src/libespeak-ng/ssml.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libespeak-ng/ssml.c b/src/libespeak-ng/ssml.c
index 5bd5da0747..c61d5624cf 100644
--- a/src/libespeak-ng/ssml.c
+++ b/src/libespeak-ng/ssml.c
@@ -571,7 +571,7 @@ int ProcessSsmlTag(wchar_t *xml_buf, char *outbuf, int *outix, int n_outbuf, con
 	char *uri;
 	int param_type;
 	char tag_name[40];
-	char buf[80];
+	char buf[160];
 	PARAM_STACK *sp;
 	SSML_STACK *ssml_sp;
 
